src/components/Router/index.js

```diff
@@ -83,6 +92,7 @@ class MainRouter extends Component {
 
   render = (props) =>
     <HistoryRouter {...props}
+      steps={props.options.steps}
```

Comment:
Do you know why github is showing these changes that have already been merged in a separate PR?

Reply:
No, I thought there was some invisible character or indentation change but I can't see it...